_tarsnap

```diff
@@ -0,0 +1,163 @@
+#compdef tarsnap
```

Comment:
Could this file be put into `misc/zsh_completion/_tarsnap` ?  Or some other filename, but definitely within `misc/` and beginning with `zsh`.